chart/values.yaml

```
@@ -220,6 +220,21 @@ secret: []
 #   secretName: ""
 #   secretKey: ""
 
+# Enables selected built-in secrets that are set via environment variables by default.
+# Those secrets are provided by the Helm Chart secrets by default but in some cases you
+# might want to provide some of those variables with _CMD or _SECRET variable, and you should
+# in this case disable setting of those variables by setting the relevant configuration to false.
+enableBuiltInSecretVariables:
```

```suggestion
enableBuiltInSecretEnvVars:
```

Still on the fence with this name. I'll see if I can come up with something better.